packages/i18n/src/resources/en.json

```diff
@@ -174,5 +176,11 @@
   "Cloud Workspace Description": "All data will be synchronized and saved to the AFFiNE account <1>{{email}}</1>",
   "Download data Description1": "It takes up more space on your device.",
   "Download data Description2": "It takes up little space on your device.",
-  "Enabled success": "Enabled success"
+  "Enabled success": "Enabled success",
+  "Export AFFINE backup file (coming soon)": "Export AFFINE backup file (coming soon)",
+  "Data sync mode": "Data sync mode",
+  "Download all data": "Download all data",
+  "It takes up more space on your device.": "It takes up more space on your device.",
+  "Download core data": "Download core data",
+  "It takes up little space on your device.": "It takes up little space on your device."
```

Comment:
These seem to already exist.
Their keys are：
"Download data",
"Download data Description1",
"Download data Description2".